packages/core/saved-objects/core-saved-objects-server-internal/src/saved_objects_service.ts

```diff
@@ -266,6 +266,14 @@ export class SavedObjectsService
         take(1)
       ).toPromise();
 
+      this.logger.info('!!!!!!!About to start migrations!!!!!!!');
+      // Pretend to be busy...
+      await new Promise((res) => {
+        setTimeout(res, 10_000);
```

Comment:
Can we also try: 
 - 5mins (still expect to see "failed to start after [4] attempts")
 - 50mins (expect cloud to fail upgrade after 20/30 mins depending on region but probably a different error than "failed to start after [4] attempts" because in this case we're not bootlooping)